### DISCUSSION TITLE
Fansly auth help

### DISCUSSION THREAD

**Discussion by iStruggles** [DIGITALCRIMINALS/OnlyFans#580] [Q&A] [answered]
I was able to auth with Onlyfans just fine, but I'm a little confused where to grab the auth_id, sess, and x_bc for Fansly. Any help would be greatly appreciated.

**Comment by purpleemoji** [comment, marked as answer]
Log into Fansly > open network tools> refresh page > filter results for 'account?ids=' (no quotes) > Click on first one > Scroll down to 'Request Headers' section, Here you will find your 'Authorization' and 'user_agent', 'sess' and 'x_bc'

Unfortunately, there are other issues with scraping from fansly with the current commit.

**Reply by iStruggles** [reply]
Many thanks, this was exactly what I needed! By chance, is start_ofd.py crashing immediately after successfully authenticating one of those "other issues" you mentioned?
